backend/homework/7/src/main/java/org/example/functions/LowestPriceVehicle.java

```diff
@@ -0,0 +1,25 @@
+package org.example.functions;
+
+import org.example.entity.Vehicle;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Comparator;
+import java.util.List;
+
+@Component
+public class LowestPriceVehicle {
+
+
+    public final List<Vehicle> vehicles;
+
+    @Autowired
+    public LowestPriceVehicle(List<Vehicle> vehicles) {
+        this.vehicles = vehicles;
+    }
+    public Vehicle findLowestPriceVehicle(List<Vehicle> vehicles) {
+        return vehicles.stream()
```

Comment:
it is possible that the object will be null, so it's important to provide a proper message or handling mechanism for that scenario.